### PR TITLE
Fix some typos, etc in the docs

### PR DESCRIPTION
**Pull request by olgavrou**
Also we say in some places that a node `can't join the network` until the members accept it, but we start by making a node join the network and then having members vote for it, so replaced those mentions with `node can't participate in the network`

### CODE REVIEW

File: sphinx/source/concepts.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ Network and Nodes
 
 A CCF network consists of several nodes, each running on top of a Trusted Execution Environment (:term:`TEE`), or enclave, such as :term:`SGX`. As such, a CCF network is decentralised and highly-available.
 
-Nodes are run and maintained by :term:`operators`. However, nodes must be trusted by the consortium of members before joining a CCF network.
+Nodes are run and maintained by :term:`operators`. However, nodes must be trusted by the consortium of members before participating in a CCF network.
```
Comment (jumaffre) [inline sphinx/source/concepts.rst]: I think that `joining` is fine. `Participating` involves that they are active in the consensus but we could imagine a scenario where nodes join first and only participate once they have caught up.

Comment (jumaffre) [inline sphinx/source/concepts.rst]: I think that `joining` is fine. `Participating` involves that they are active in the consensus but we could imagine a scenario where nodes join first and only participate once they have caught up.

Comment (jumaffre) [inline sphinx/source/concepts.rst]: I think that `joining` is fine. `Participating` involves that they are active in the consensus but we could imagine a scenario where nodes join first and only participate once they have caught up.

Comment (olgavrou) [reply sphinx/source/concepts.rst]: As it is it implies that there is a step that needs to happen first by the members before a node can join the network, where as we first start up a node with `join` and then ask to be accepted by the members. Maybe not `participating` but something else?

File: sphinx/source/members/common_member_operations.rst [hunk 2]
```diff
@@ -107,7 +107,7 @@ Once the proposal to recover the network has passed under the rules of the :term
             Node 3-->>Users: Response
         end
 
-Once the recovery of the private ledger on all the nodes that have joined the new network is complete, the ledger is fully recovered and users are able to continue issuing business transactions.
+Once the recovery of the private ledger is complete on all the nodes that have joined the new network, the ledger is fully recovered and users are able to continue issuing business transactions.
```
Comment (achamayou) [inline sphinx/source/members/common_member_operations.rst]: This is inaccurate, the recovery of the private ledger only needs to have globally committed/completed on a consensus quorum of nodes (f+1 for raft and 2f+1 for pbft).

Comment (olgavrou) [reply sphinx/source/members/common_member_operations.rst]: I'll update it

File: sphinx/source/concepts.rst [hunk 3]
```diff
@@ -35,8 +35,8 @@ The integrity of the ledger is guaranteed by a Merkle tree whose root is periodi
 Governance
 ----------
 
-A CCF network is governed by a consortium of :term:`members`. The :term:`constitution`, recorded in the ledger itself, defines a set of rules that members should follow.
+A CCF network is governed by a consortium of :term:`members`. The :term:`constitution`, recorded in the ledger itself, defines a set of rules that members must follow.
 
-Members can submit proposals to modify the state of the Key-Value Store. For examples, members can decide to allow a new trusted user to issue requests to the application or to add a new member to the consortium.
+Members can submit proposals to modify the state of the Key-Value Store. For example, members can decide to allow a new trusted user to issue requests to the application or to add a new member to the consortium.
```
Comment (ashamis) [inline sphinx/source/concepts.rst]: please replace decide with vote

**Pull request merged**